lms/resources/_js_config/__init__.py

```diff
@@ -497,7 +497,11 @@ def _get_product_info(self):
 
         if self._request.product.settings.groups_enabled:
             product_info["api"]["listGroupSets"] = {
-                "authUrl": self._request.route_url(product.route.oauth2_authorize),
+                "authUrl": (
+                    self._request.route_url(product.route.oauth2_authorize)
```

Comment:
If I remember correctly, `None` gets JSON-serialized as `null`.

The type definition in client is right now stating the property is a string, but is possible that it is not defined at all. See https://github.com/hypothesis/lms/blob/a61cb78b190f0d39e8d0706cd62568c66dd64407/lms/static/scripts/frontend_apps/config.ts#L21

If it can be null now, we should change that type definition to `authUrl?: string | null`, so that the code knows that, when the property is present, it can be a string or null.

That said, handling an undefined prop or a key with value null is similar in JavaScript, so maybe that's why it works. But it's not exactly the same and we should change it, but should be ok to change on a follow-up PR